packages/component/src/Activity/Bubble.js

```diff
@@ -82,21 +88,27 @@ const Bubble = ({ 'aria-hidden': ariaHidden, children, className, fromUser, nub
         nubSize: bubbleNubSize,
         side: 'bot'
       };
-  const [direction] = useDirection();
 
   return (
     <div
       aria-hidden={ariaHidden}
       className={classNames(
+        'webchat__bubble',
         ROOT_CSS + '',
-        direction === 'rtl' ? 'webchat__bubble--rtl' : '',
         bubbleStyleSet + '',
-        { 'from-user': fromUser, webchat__bubble_has_nub: nub },
+        {
+          'webchat__bubble--from-user': fromUser,
+          'webchat__bubble--hide-nub': nub !== true && nub !== false,
```

Comment:
Nub (and avatar) has 3 different setting:

- No nubs
- Show nub
- Hide nub

Hide means, we need to preserve the space for nub, but we are not showing the nub. This is different than no nub, which means no space need to be preserve because there are no nubs.

Nub can be hidden because of grouping, for example, the following red circle is "hide nub". This is different than "no nub" because a space is reserved there. Otherwise, without preserving the space, the bubble next to the red circle will be moved a bit to the right and won't align well.

![image](https://user-images.githubusercontent.com/1622400/88976359-c0245c80-d270-11ea-9ccd-fc95a47c152c.png)

And the following red circles are "no nub", there are no space preserved for the nub.

![image](https://user-images.githubusercontent.com/1622400/88976416-daf6d100-d270-11ea-9e87-f0a12cbd25ef.png)

Do you mean?

- `webchat__bubble--nub-visibility-collapse`
- `webchat__bubble--nub-visibility-visible`
- `webchat__bubble--nub-visibility-hidden`